src/components/MenuPanel/MenuPanel.js

```diff
@@ -105,8 +120,52 @@ class MenuPanel extends React.PureComponent {
                     : this.renderAppGroup(app, false)
                 )}
               </div>
-              <h1 style={{ marginTop: '24px' }}>System</h1>
-              <div>{systemApps.map(app => this.renderAppGroup(app, true))}</div>
+              <FocusVisible>
+                {({ onFocus, focusVisible }) => (
+                  <StyledButton
+                    onFocus={onFocus}
+                    focusVisible={focusVisible}
+                    onClick={this.handleToggleSystemApps}
+                  >
+                    <h1
+                      style={{
+                        marginTop: '24px',
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        alignItems: 'flex-end',
+                      }}
+                    >
+                      <span>System</span>
+                      <span
+                        css={`
+                          font-size: 7px;
+                          opacity: 0.7;
+                        `}
+                      >
+                        {systemAppsOpened ? '▼' : '▲'}
```

Comment:
Can we use this icon instead, for consistency? https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/src/components/assets/arrow.svg

Reply:
Sure, you are the one that suggested the arrow in the first place.

Reply:
I know 😅 I should have been more explicit: it was meant to describe the icon, not to be used literally.